### PR TITLE
[Bug](information-schema) fix some bug of information_schema.PROCESSLIST

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
1. information_schema.PROCESSLIST's schema not matched with FrontendServiceImpl::showProcessList
2. rpc failed on when cloudCluster is null
3. SchemaScanner::fill_dest_column_for_range use uint32_t to store datetimev2 data
```cpp
case TYPE_DATETIMEV2: {
            uint32_t num = *reinterpret_cast<uint64_t*>(data);
```

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/exec/schema_scanner/schema_processlist_scanner.cpp [hunk 1]
```diff
@@ -90,48 +97,36 @@ Status SchemaProcessListScanner::_fill_block_impl(vectorized::Block* block) {
...
         for (size_t row_idx = 0; row_idx < row_num; ++row_idx) {
             const auto& row = process_list[row_idx];
+            if (row.size() != _s_processlist_columns.size()) {
+                return Status::InternalError(
+                        "process list meet invalid schema, schema_size={}, input_data_size={}",
+                        _s_processlist_columns.size(), row.size());
+            }
 
             // Fetch and store the column value based on its index
             std::string& column_value =
```
Comment (HappenLee) [inline be/src/exec/schema_scanner/schema_processlist_scanner.cpp:107]: seems do a unless code

File: regression-test/suites/external_table_p0/info_schema_db/test_info_schema_db.groovy [hunk 2]
```diff
@@ -131,4 +131,6 @@ suite("test_info_schema_db", "p0,external,hive,external_docker,external_docker_h
     qt_sql116 """select table_catalog, table_schema, table_name from information_schema.tables where table_schema='${innerdb}'"""
     qt_sql117 """select table_catalog, table_schema, table_name from ${catalog_name}.information_schema.columns where table_schema='tpch1_parquet'"""
     qt_sql118 """select table_catalog, table_schema, table_name from ${catalog_name}.INFORMATION_SCHEMA.COLUMNS where TABLE_SCHEMA='tpch1_parquet'"""
+
+    sql "select * from information_schema.PROCESSLIST;"
```
Comment (HappenLee) [inline regression-test/suites/external_table_p0/info_schema_db/test_info_schema_db.groovy:135]: why only case not `.out` result ?

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by Mryange — APPROVED** [review]

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**